apps/api/src/domain/domainModelValidators/errors/context/InvalidCharRangeError.ts

```diff
@@ -0,0 +1,11 @@
+import { InternalError } from 'apps/api/src/lib/errors/InternalError';
+
+const formatNumericRange = ([x, y]: [number, number]): string => `[${x},${y}]`;
+
+export default class InvalidCharRangeError extends InternalError {
+    constructor(charRange: [number, number]) {
```

Comment:
Maybe in the future this could also give which case failed (ie., negative start index, start > end, etc.)

Reply:
This is a good point. I should propagate through some `InnerErrors` here. I will either do this now or make a story.

Reply:
Added a [story](https://www.pivotaltracker.com/story/show/181894276)